docs/overview/versioning-policy.md

```diff
@@ -166,10 +166,7 @@ following schedule for determining the [Node.js releases](https://nodejs.org/en/
 
 - At any given point in time we support exactly two adjacent even-numbered
   releases of Node.js, for example v12 and v14.
-- Once a new Node.js release becomes _Active LTS_ we switch to support that
-  release and the previous one. The switch is not immediate but done as soon
-  as possible. You can find the Node.js version supported by each release
-  in the `engines` field in the root `package.json` of a new app.
+- Once a new Node.js release becomes _Active LTS_ we switch to support that release and the previous one. The switch is not immediate but done as soon as possible. This support will come in the first Backstage main release line release after a Node.js release becomes _Active LTS_. You can find the Node.js version supported by each release in the `engines` field in the root `package.json` of a new app.
```

Comment:
> This support will come in the first Backstage main release line release after a Node.js release becomes _Active LTS_.

This isn't necessarily true since it depends on the timing of things. If there are unresolved blocking issues or the releases fall very close to each other we make exceptions. I'd prefer to keep this as a best effort with an intended target.

I think a fairly big improvement could actually be to simply list the current supported versions here? Now that we have versioned docs for stable/next releases we can make that update properly.

Reply:
Ok, cool. I can do that, in the past we've tried to avoid it as it means another place we need to make sure to update but you are right, now that things are versioned it's a bit easier.

Reply:
Yeah that sounds great @awanlin and @Rugvip

Reply:
That's great. Listing the supported versions and linking this page in the Getting Started page would be perfect.